docs/docsource/installation/uninstall/index.rst

```diff
@@ -1,11 +1,19 @@
 Uninstall
 =========
 
-To uninstall/delete the “my-zenko” deployment:
+To uninstall/delete the “my-zenko” deployment, run:
 
-.. code:: bash
+::
 
-   $ helm delete my-zenko
+  $ kubectl delete cosmos --all
 
-The command removes all Kubernetes components associated with the
-chart and deletes the deployed instance.
\ No newline at end of file
+This deletes all Cosmos objects from Kubernetes. 
+   
+Then, run:
+
+::
+  
+  $ helm delete my-zenko
```

Comment:
might want to add `--purge` otherwise you'll get an error about an already existing deployment if you try to redeploy using the same release name

Reply:
Resolved in the force-push.

Reply:
I ran into this last week, that's true.

Reply:
I fixed this wrong. 

To be clear: the ``--purge`` option goes to the Helm command, and not the ``kubectl delete`` command